infra/src/webapp.tf

```diff
@@ -156,10 +156,10 @@ locals {
     JIRA_ORGANIZATION_NAME_CUSTOM_FIELD = var.jira_organization_name_custom_field
 
     # Apim connection
-    AZURE_APIM                                   = var.azure_apim
-    AZURE_APIM_RESOURCE_GROUP                    = var.azure_apim_resource_group
-    AZURE_APIM_DEFAULT_SUBSCRIPTION_PRODUCT_NAME = "io-services-api" # FIXME: create and use variable
-    AZURE_SUBSCRIPTION_ID                        = data.azurerm_subscription.current.subscription_id
+    AZURE_APIM                           = var.azure_apim
+    AZURE_APIM_RESOURCE_GROUP            = var.azure_apim_resource_group
+    AZURE_APIM_SUBSCRIPTION_PRODUCT_NAME = "io-services-api" # FIXME: create and use variable
```

Comment:
TIP: Since you are editing this part it would be a good thing to solve the FIXME by creating a proper variable

Reply:
[d4c0e96](https://github.com/pagopa/io-services-cms/pull/144/commits/d4c0e968be38f2109cf739a7a7982db28fe1e7ea)

Reply:
[01f22d3](https://github.com/pagopa/io-services-cms/pull/144/commits/01f22d3e277e9af7caa9b4efd6690aa35484431c)